cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -137,6 +138,8 @@
 
   private final PreferencesStore preferencesStore;
 
+  private final ConsoleSettingsStore consoleSettingsStore;
```

Comment:
This class is for v2 APIs right? Should you be able to access a v3 feature (Console settings) through it? Maybe instead we should add a v3 unrecoverable/reset API and only add console settings deletion for now?